Remove FAQ and fix links (#86)
ezsystems/user-documentation: docs/personalization/recommendation_models.md

# 6. Recommendation Models

> Chapter [2. Use Cases](use_cases.md) describes where recommendations can be used in real-world situations. They are based on algorithms and the calculated result of an algorithm is called a model. The models work in the background and are updated regularly to provide recommendations. There are several types of models available:

Reviewer: ```suggestion
Chapter [2. Use Cases](use_cases.md) describes real-life situations in which you could use recommendations. 
Recommendations are based on algorithms and the calculated result of an algorithm is called a model. 
Models work in the background and are updated regularly to provide recommendations. 
There are several types of models available:
```
